exporter/sentinels.go

```diff
@@ -66,6 +66,16 @@ func (e *Exporter) extractSentinelMetrics(ch chan<- prometheus.Metric, c redis.C
 		}
 		masterAddr := masterIp + ":" + masterPort
 
+		masterCkquorum, err := redis.String(doRedisCmd(c, "SENTINEL", "CKQUORUM", masterName))
+		log.Debugf("Sentinel ckquorum status for master %s", masterName)
```

Comment:
I think this log message is in the wrong place or not capturing enough info. Should this output `masterCkquorum` and `err` as well?

Reply:
Hi, sorry for the late response. Oh yes, I think you're right. I will update it soon